content/includes/breadcrumbs.html

```diff
@@ -1,39 +1,33 @@
-{% if gatePage === true %}
-  {% assign gatePageClassName = 'va-nav-breadcrumbs--gate' %}
-{% endif %}
+{% if gatePage === true %} {% assign gatePageClassName = 'va-nav-breadcrumbs--gate' %} {% endif %}
 
-<nav class="va-nav-breadcrumbs {{gatePageClassName}}">
-  <ul class="row va-nav-breadcrumbs-list columns" role="menubar" aria-label="Primary">
-    <li><a href="/">Home</a></li>
-      {% assign current_page = breadcrumb_path | last %}
-      {% for crumb in breadcrumb_path %}
-        {% if current_page.name != crumb.name %}
-          <!-- removed to remove 'active' breadcrumb -->
-        
-          {% comment %}
-            Each breadcrumb item has a child object. One of the properties of
-            that object is `file` which loads the front matter metadata from
-            the corresponding Markdown input file.
+<nav aria-label="Breadcrumb" aria-live="polite" aria-relevant="additions text" class="va-nav-breadcrumbs js-visual {{gatePageClassName}}"
+  id="va-breadcrumbs">
+  <p class="usa-sr-only">Breadcrumb navigation will usually show all page links. It will adjust to show only the previous page when zoomed in, or
+    viewed on a mobile device.</p>
+  <ol class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
+    <li>
+      <a href="/">Home</a>
+    </li>
+    {% assign current_page = breadcrumb_path | last %} {% for crumb in breadcrumb_path %}
 
-            Below, we’ve assigned that object to a variable named
-            'crumb_child_meta'.
-
-            If these templates are still using tinyliquid, you can
-            dump the properties of this object using {{ crumb | keys }}.
-          {% endcomment %}
-
-          {% assign crumb_child_meta = crumb.children.0.file %}
-          <li>
-            <a href="/{{ crumb.path }}">
-              {% if crumb_child_meta.display_title %}
-                {{ crumb_child_meta.display_title }}
-              {% elsif crumb_child_meta.title %}
-                {{ crumb_child_meta.title }}
-              {% else %}
-                {{ crumb.name | replace: '-', ' ' | capitalize }}
-              {% endif %}
-            </a></li>
-        {% endif %}
-      {% endfor %}
-  </ul>
-</nav>
+    <!-- Adding back in the last link as aria-current="page" -->
+    {% comment %} Each breadcrumb item has a child object. One of the properties of that object is `file` which loads the front
+    matter metadata from the corresponding Markdown input file. Below, we’ve assigned that object to a variable named 'crumb_child_meta'.
+    If these templates are still using tinyliquid, you can dump the properties of this object using {{ crumb | keys }}. {%
+    endcomment %} {% assign crumb_child_meta = crumb.children.0.file %}
```

Comment:
The formatting of this block seems kinda wonky. Can we pull that `{% assign... %}` onto its own line and bring `endcomment %}` to the same line as its `{%`?